scapy/sendrecv.py

```diff
@@ -893,7 +850,8 @@ def sniff(count=0, store=True, offline=None, prn=None, lfilter=None,
     try:
         if started_callback:
             started_callback()
-        while sniff_sockets:
+        continue_sniff = True
```

Comment:
Is that the only relevant fix? =)

Reply:
That’s for https://github.com/secdev/scapy/issues/1918

There’s a lot of unrelevant cleaning happening :smile: sorry for the messy PRs